app/models/paymentOnAccount/PaymentOnAccount.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.paymentOnAccount
+
+import models.incomeSourceDetails.TaxYear
+
+case class PaymentOnAccount(
```

Comment:
assuming we made a recently decision to move most of the 1553 processing to BE; see: https://jira.tools.tax.service.gov.uk/browse/MISUV-3817
worth to create a techDebt story to move this model to BE as well